apps/server/src/modules/learnroom/mapper/common-cartridge.mapper.spec.ts

```diff
@@ -0,0 +1,378 @@
+import { faker } from '@faker-js/faker';
+import { DeepMocked, createMock } from '@golevelup/ts-jest';
+import { MikroORM } from '@mikro-orm/core';
+import { ConfigService } from '@nestjs/config';
+import { Test, TestingModule } from '@nestjs/testing';
+import { ComponentProperties, ComponentType } from '@shared/domain/entity';
+import { courseFactory, lessonFactory, setupEntities, taskFactory, userFactory } from '@shared/testing';
+import { OmitVersion, createIdentifier } from '@src/modules/common-cartridge/utils';
+import {
+	CommonCartridgeElementProps,
+	CommonCartridgeElementType,
+	CommonCartridgeIntendedUseType,
+	CommonCartridgeOrganizationBuilderOptions,
+	CommonCartridgeResourceProps,
+	CommonCartridgeResourceType,
+	CommonCartridgeVersion,
+} from '../../common-cartridge';
+import { LearnroomConfig } from '../learnroom.config';
+import { CommonCartridgeMapper } from './common-cartridge.mapper';
+
+describe('CommonCartridgeMapper', () => {
+	let module: TestingModule;
+	let sut: CommonCartridgeMapper;
+	let orm: MikroORM;
+	let configServiceMock: DeepMocked<ConfigService<LearnroomConfig, true>>;
+
+	beforeAll(async () => {
+		module = await Test.createTestingModule({
+			providers: [
+				CommonCartridgeMapper,
+				{
+					provide: ConfigService,
+					useValue: createMock<ConfigService<LearnroomConfig, true>>(),
+				},
+			],
+		}).compile();
+		orm = await setupEntities();
+		sut = module.get(CommonCartridgeMapper);
+		configServiceMock = module.get(ConfigService);
+	});
+
+	afterAll(async () => {
+		await module.close();
```

Comment:
await module.close(); should also execute the orm.close() it can be removed.

Reply:
Done